chapter_multilayer-perceptrons/mlp_vn.md

```diff
@@ -506,7 +509,10 @@ In some ways, you have an advantage over anyone working the 1990s, because you c
 Previously, getting these nets training required researchers to code up thousands of lines of C and Fortran.
 -->
 
-*dịch đoạn phía trên*
+Tóm lại, bây giờ chúng ta biết cách kết hợp các hàm phi tuyến để xây dựng các mô hình mạng nơ-ron đa tầng.
+Ngoài ra, kiến thức của bạn đã giúp bạn thành thạo một bộ công cụ tương tự với một học viên vào khoảng năm 1990.
+Trong vài trường hợp khác, bạn có lợi thế hơn bất kỳ ai làm việc trong những năm 1990, bởi vì bạn có thể tận dụng triệt để nguồn mở về nền tảng học sâu để xây dựng các mô hình một cách nhanh chóng, chỉ với sử dụng một vài dòng mã.
```

Comment:
từ `in some way` mình không biết dịch là gì cho hợp nên tạm bỏ
```suggestion
Bạn còn có lợi thế hơn bất kỳ ai làm việc trong những năm 1990, bởi vì bạn có thể tận dụng triệt để các framework học sâu nguồn mở để xây dựng các mô hình một cách nhanh chóng, chỉ với một vài dòng mã.
```

Reply:
@minhduc0711 trong glossary chưa có từ framework, bạn có đề xuất dịch sang tiếng việt hay giữ nguyên?

Reply:
mình thấy trong slack thì có vẻ mọi người đã thống nhất giữ nguyên rồi thì phải, để lát mình thêm từ này vào glossary luôn

Reply:
`in some way` mình thấy có thể dịch là `Xét theo một khía cạnh nào đó, bạn còn có lợi thế...`

Reply:
@rootonchair thanks, mình đã edit